src/main/java/storage/Storage.java

```diff
@@ -0,0 +1,156 @@
+package storage;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.ArrayList;
+import java.util.Scanner;
+import java.util.stream.Collectors;
+import task.Task;
+import task.Todo;
+import task.Deadline;
+import task.Event;
```

Comment:
Maybe put these before the other imports and after the package statement
```suggestion
package storage;

import task.Task;
import task.Todo;
import task.Deadline;
import task.Event;
```